overlord/devicestate/devicestate_remodel_test.go

```diff
@@ -1039,6 +1051,37 @@ func (s *deviceMgrRemodelSuite) TestRemodelReregLocalFails(c *C) {
 	c.Assert(chg, IsNil)
 }
 
+func installSnap(c *C, st *state.State, yaml string, si *snap.SideInfo) *snap.Info {
+	info := snaptest.MakeSnapFileAndDir(c, yaml, nil, si)
+	snapstate.Set(st, info.InstanceName(), &snapstate.SnapState{
+		SnapType: string(info.Type()),
+		Active:   true,
+		Sequence: snapstatetest.NewSequenceFromSnapSideInfos([]*snap.SideInfo{&info.SideInfo}),
+		Current:  info.Revision,
+	})
+	return info
+}
+
+func installEssentialSnaps(c *C, st *state.State, base string) {
```

Comment:
Maybe the code of the `install*` functions here and in `manager_test.go` could be shared? snaptest could be the container package I think.

Reply:
I could see `installSnap` being moved there, but it feels like `installEssentialSnaps` has a pretty specific use case to these tests. Especially since the one in `managers_test.go` takes in a `bootloader.Bootloader`. Thoughts?

Reply:
Maybe a new package called `overlordtest`? @pedronis , what's your opinion on this?

Reply:
can't this go into already existing snapstatetest ?